testData/psi/Docstrings.smk

```diff
@@ -0,0 +1,4 @@
+rule NAME: """Docstring 1"""
```

Comment:
I think several docstrings is not the best example and probably a side-effect of  snakemake implementation. So I suggest use several tests for this feature. E.g. 2 real life use-cases, e.g. 1) single line doc string 2) multiline doc string. And a couple of syntetic usecases e.g. 3) for non """ quotes and 4) for some crazy things like several docstrings, docstrings starting on same line as rule, e.tc.